src/main/java/com/autotune/utils/KruizeSupportedTypes.java

```diff
@@ -78,7 +78,7 @@ private KruizeSupportedTypes() { }
 			new HashSet<>(Arrays.asList("deployment", "pod", "container"));
 	
   public static final Set<String> SUPPORTED_FORMATS =
-			new HashSet<>(Arrays.asList("cores", "MiB"));
+			new HashSet<>(Arrays.asList("cores", "Bytes", "KiB", "MiB", "GiB"));
```

Comment:
Please take a look at the supported kubernetes resource units as per the official documentation
https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/

We need to support all of the above

Reply:
Hi @dinogun , 

As per the documentation, I have added the all missing formats and modified the PR. Please let me know your comments. 

Thanks

Reply:
Hi @dinogun, 

Updated the code for all supported formats. Please review again.